consensus/types/state.go

```diff
@@ -52,7 +52,7 @@ func (rs RoundStepType) String() string {
 //-----------------------------------------------------------------------------
 
 // RoundState defines the internal consensus state.
-// It is Immutable when returned from ConsensusState.GetRoundState()
+// It should be immutable when returned from ConsensusState.GetRoundState(), but it's not.
```

Comment:
I think we can just delete this line and the 2 lines below it.  It already says "Not thread safe".